specviz/widgets/workspace.py

```diff
@@ -320,8 +320,8 @@ def _on_load_data(self):
         """
         When the user loads a data file, this method is triggered. It provides
         a file open dialog and from the dialog attempts to create a new
-        :class:`~specutils.Spectrum1D` object and thereafter adds it to the
-        data model.
+        :class:`~specutils.SpectrumList` object and thereafter adds the
+        contents to the data model.
```

Comment:
It's not adding `SpectrumList` to the data model -- it's slicing the `SpectrumList` to get `Spectrum1D`s, and adding them to the data model. May want to just clarify that?

Reply:
Yep I think this may have been the result of a blind search-and-replace.